### ISSUE TITLE
add 'drumkit edited' indication

### ISSUE THREAD

**Issue by thijz** [hydrogen-music/hydrogen#232]
add some sort of indication that the drumkit you are using has been modified

reason : when creating a song i frequently use instruments form various drumkits
i mostly start with the drumkit i think is most suited for the song, but as i'm working on the song i add new instruments, remove instruments, or move them up/down
When i'm done i can save the song and open it again without any issues

however..
when i want to move the song to an other pc there is no guarantee that the song will play correctly
to avoid this i normally save the new drumkit i created for that song as a new drumkit (specifically for that song) and when i move the song to an other pc i also export/import the new drumkit

sometimes i forget to save the drumkit after i modified it ...

Bottom line : it woul be nice to have some sort of visual indication of the modified-and-unsaved drumkit state
Maybe a 'do you want to save the modified drumkit?' popup would also bee a good idea

i feel that a lot of confusion among the hydrogen users is caused by the fact that saving a song is not enough if you want to port it to an other pc
adding this indication wont fix that, but it should at least make clear to the user that he/she is not using a standard drumkit

**Comment by thijz** [comment]
related to #130

**Comment by theGreatWhiteShark** [comment]
Hmm. I know where you are heading but I don't like the idea too much. The idea of a drumkit is to provide a default setting of parameters and all samples required. All customization are directly stored in the .h2song file and for sets of parameters used by many songs, the tweaked version of the drumkit can be saved into a new one. When opening a song on a different computer all drumkits (specifically their samples) you used in that song have to be present too. 

Creating custom drumkits by combining instruments of different ones or even creating new ones is something for advanced users. Regular ones would probably mistake such an indication for being required to create a new or overwrite an existing drumkit what would generate a number of new problems.

I would suggest to not implemented this and to proceed your idea in #272 instead which feels more fitting for this use case.
